package.json

```diff
@@ -43,7 +43,7 @@
     "@types/node": "^14.14.16",
     "@typescript-eslint/eslint-plugin": "^5.43.0",
     "@typescript-eslint/parser": "^5.43.0",
-    "aws-sdk": "^2.818.0",
+    "@aws-sdk/lib-dynamodb": "^3.287.0",
```

Comment:
Hey, I was thinking, wouldn't it be better if we install the AWS SDK version provided by the [AWS Lambda Node.js.18 Runtime](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html) (v3.188.0)? Many users might want to keep their bundle size low by relying on the provided SDK version. So, my suggestion is to run the unit tests using that SDK version. What are your thoughts on this?

Reply:
that's a negative ghost rider. lambda is only one of a myriad of possible platforms this would be used on. additionally, if you need to override or assert a particular aws-sdk version you can use a package manager that supports that feature, and for bundling you can specify that aws-sdk is external to the bundle (which you should be doing already anyhow)

Reply:
However, based on the changes in [@aws-sdk/lib-dynamodb](https://github.com/aws/aws-sdk-js-v3/commits/main/lib/lib-dynamodb), this would most likely not be an issue.